neo/rawio/intanrawio.py

```diff
@@ -434,14 +453,29 @@ def read_variable_header(f, header):
     ("electrode_impedance_phase", "float32"),
 ]
 
+stream_type_to_name_rhs= {
```

Comment:
@h-mayorquin,

this is more accurate because the stream_ids change between rhs and rhd so we should translate correctly. `0` is still always amplifier but rhs has extra analog so we could force it to have a random number (like 12) if you think it will be really disruptive on the neuro-conv side...

Reply:
I don't understand,  shoulnd't we have a one dict for rhd and another for rhs if they don't have consistent stream names?

Reply:
Yes I was saying we SHOULD have two different. My concern was that the old behavior was to use the RHD dict for the rhs files and we were actually messing up the streams. I was just worried that someone may have written an Interface on Neuroconv expecting this behavior so I wanted to double check!

Reply:
Ok, I agree.

Concerning neuroconv I don't think we did.